Clarify the secondary indexes usage in the concepts article (#10855)
ydb-platform/ydb: ydb/docs/en/core/concepts/_includes/secondary_indexes.md

- Deleted from an existing table with the YQL [`ALTER TABLE`](../../yql/reference/syntax/alter_table/index.md) statement or the YDB CLI [`table index drop`](../../reference/ydb-cli/commands/secondary_index.md#drop) command.
- Deleted together with the table using the YQL [`DROP TABLE`](../../yql/reference/syntax/drop_table.md) statement or the YDB CLI `table drop` command.

## Using secondary indexes {#use}

> To get detailed information on the secondary indexes usage in applications refer to [this article](../../dev/secondary-indexes.md) in the Development section of the documentation.

Reviewer: ```suggestion
For detailed information on using secondary indexes in applications, refer to the [relevant article](../../dev/secondary-indexes.md) in the documentation section for developers.
```
